silence/silence.go

```diff
@@ -1053,3 +1067,35 @@ func openReplace(filename string) (*replaceFile, error) {
 	}
 	return rf, nil
 }
+
+// logging silence status changes.
+//
+// XXX: This rewrites some code present in cli/format/format.go,
+// labelsMatcher() and cli/format/format_extended.go,
+// FormatSilences(). Refactoring this seems a little out of scope for
+// now.
+func (s *Silences) logginSilences(msg string, sil *pb.Silence) {
```

Comment:
Typo?

```suggestion
func (s *Silences) loggingSilences(msg string, sil *pb.Silence) {
```

Reply:
i renamed this to logSilence, and rebased everything.